README.md

```diff
@@ -77,6 +77,26 @@ progressively.init()
 
 See [demo](https://thinker3197.github.io/progressively) for examples.
 
+### Use medium quality on small screens
+
+You can add a medium version via ```data-progressive-sm``` to reduce the filesize on mobile devices with small screens.
```

Comment:
For inline code use just a [single backtick](https://guides.github.com/features/mastering-markdown/#syntax).